core/src/main/java/io/undertow/server/handlers/AllowedMethodsHandler.java

```diff
@@ -31,7 +31,7 @@
 import io.undertow.httpcore.StatusCodes;
 
 /**
- * Handler that whitelists certain HTTP methods. Only requests with a method in
+ * Handler that allowlists certain HTTP methods. Only requests with a method in
```

Comment:
From the peanut gallery: Perhaps more natural:
```suggestion
 * Handler that allows certain HTTP methods. Only requests with a method in
```
(same for disallow below)

I appreciate the cleanup, thanks all!

Reply:
I agree, much better!

Reply:
I've updated, with your suggested change.